static/js/services/enketo.js

```diff
@@ -86,22 +86,24 @@ angular.module('inboxServices').service('Enketo',
       });
     };
 
-    var translateXml = function(text, title, language) {
+    var translateXml = function(text, language, title) {
```

Comment:
Do we need to worry about the callers here if we're changing the signature of the function?